### PR TITLE
docs(l10n/html): update manuals

### PR DESCRIPTION
**Pull request by miurahr**
## Pull request type

Please mark github LABEL of the type of change your PR introduces:

- Bug fix -> [bug]
- Feature enhancement -> [enhancement]
- Documentation -> [documentation]
- Build and release changes -> [build/release]
- Other (describe below)

## Which ticket is resolved?

- Title uplicate file `docs/XX/omegat.css` on windows
  - Bugs: https://sourceforge.net/p/omegat/bugs/1148
 
## What does this PR change?

- Update manuals html
- Remove old omegat.css
- Fix bugs#1148

### CODE REVIEW

File: docs/be/version_be.properties [hunk 1]
```diff
@@ -1,2 +1,2 @@
 # NOI18N
-version=4.3.1
+version=4.2.0
```
Comment (miurahr) [inline docs/be/version_be.properties:2]: something degraded?

Comment (miurahr) [reply docs/be/version_be.properties:2]: This was modified in 290b6672

Comment (miurahr) [reply docs/be/version_be.properties:2]: @brandelune  is it intended change?

Comment (brandelune) [reply docs/be/version_be.properties:2]: Yes. The version that appears in the manual is always 4.2. There was no 4.3.1 version.

Comment (miurahr) [reply docs/be/version_be.properties]: A commit message did not said fixes you explains here, so looks doubt...

Comment (brandelune) [reply docs/be/version_be.properties:2]: Sorry. I wanted to make sure that the online manual version numbers did not confuse the users. This situation will be fixed as more manuals move to 6.0.

**Comment by miurahr** [comment]
merged into master.

**Pull request closed without merging**